NoObjections/core/src/com/missionbit/game/states/MenuState.java

```diff
@@ -19,12 +20,18 @@
     public MenuState(GameStateManager gsm) {
         super(gsm);
         menu = new Texture("coolbg1.png");
+        playBtn = new Texture("playbutton.png");
     }
 
     @Override
     public void handleInput() {
         if(Gdx.input.justTouched()){
-            gsm.set(new PlayState(gsm));
+            if(Gdx.input.getX() > 483 && Gdx.input.getX() < 600){
+                    if (Gdx.input.getY() > 300 && Gdx.input.getY()<500){
+                        gsm.set(new PlayState(gsm));
+            }
```

Comment:
You can select blocks of code in Android Studio and auto-indent them using this hotkey.
It will make your nested code easier to read and less prone for errors in the future:
https://stackoverflow.com/questions/16580171/code-formatting-shortcut-in-android-studio